WORKSPACE

```diff
@@ -1,35 +1,19 @@
 workspace(name = "jsonnet")
 
-# This local_repository looks silly but it makes io_bazel_rules_jsonnet use
-# _this_ jsonnet, not another downloaded copy.
-local_repository(
-    name = "jsonnet",
-    path = ".",
-)
+load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
 
 git_repository(
     name = "io_bazel_rules_jsonnet",
     commit = "09ec18db5b9ad3129810f5f0ccc86363a8bfb6be",
     remote = "https://github.com/bazelbuild/rules_jsonnet.git",
 )
 
-new_git_repository(
+git_repository(
     name = "com_google_googletest",
     remote = "https://github.com/google/googletest.git",
 
     # If updating googletest version, also update CMakeLists.txt.in.
-    tag = "release-1.8.0",
-    build_file = "gmock.BUILD",
-)
-
-bind(
-    name = "googletest",
-    actual = "@com_google_googletest//:googletest_no_main",
-)
-
-bind(
-    name = "googletest_main",
-    actual = "@com_google_googletest//:googletest",
+    tag = "release-1.8.1",
```

Comment:
For here I am getting from bazel 0.24: `Rule 'com_google_googletest' indicated that a canonical reproducible form can be obtained by modifying arguments commit = "2fe3bd994b3189899d93f1d5a881e725e046fdc2", shallow_since = "1535728917 -0400" and dropping ["tag"]`

so would also be great to see this updated, and maybe have a comment that this is release 1.8.1